### ISSUE TITLE
Update getSettings docs with link to cascading configuration model description

### ISSUE THREAD

**Issue by wojciechczerniak** [handsontable/docs#12]
Method documentation for `getSettings` (https://handsontable.com/docs/Core.html#getSettings) should mention cascading configuration model or link (https://handsontable.com/docs/6.2.0/tutorial-setting-options.html#page-cascading) to avoid confusion ie. https://github.com/handsontable/handsontable/issues/4417

**Comment by aaronbeall** [comment]
@wojciechczerniak Is there any documentation on which options are grid, column, and cell level properties? My reading of the documentation doesn't make this clear... I know that almost every option can be used a almost any level, but there are some exceptions, for example `columns` is only a grid property, and `data` means something different at the grid level vs the column level...

**Comment by wojciechczerniak** [comment]
No. That's the main point of this issue. It's not always clear where belong each option and how they can vary in different context. Currently you have to dig in deeper into documentation and examples for features and plugins.

**Comment by AMBudnik** [comment]
Relates to https://github.com/handsontable/docs/issues/13
